src/Converter/NullAndTrimStringConverter.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Kros.KORM.Properties;
+using System;
+
+namespace Kros.KORM.Converter
+{
+    /// <summary>
+    /// <para>
+    /// Converter for string values. Based on settings, it can convert <see langword="null"/> and <see cref="DBNull.Value"/>
+    /// values to empty string and trim string value.
+    /// </para>
+    /// <para>
+    /// The converter cannot be directly instantiated. Instead of that, there are predefined static instances
+    /// <see cref="ConvertNull"/>, <see cref="TrimString"/> and <see cref="ConvertNullAndTrimString"/>.
+    /// </para>
+    /// <para>
+    /// Converter can be used for example to ensure, that value going into the database will be empty string if the column
+    /// is set <c>NOT NULL</c>.
+    /// </para>
+    /// </summary>
+    public class NullAndTrimStringConverter : IConverter
+    {
+        /// <summary>
+        /// Instance of the converter with settings:
+        /// <see cref="ConvertNullValue"/> is <see langword="true"/>, <see cref="TrimStringValue"/> is <see langword="false"/>.
+        /// </summary>
+        public static readonly NullAndTrimStringConverter ConvertNull = new NullAndTrimStringConverter(true, false);
+
+        /// <summary>
+        /// Instance of the converter with settings:
+        /// <see cref="ConvertNullValue"/> is <see langword="false"/>, <see cref="TrimStringValue"/> is <see langword="true"/>.
+        /// </summary>
+        public static readonly NullAndTrimStringConverter TrimString = new NullAndTrimStringConverter(false, true);
+
+        /// <summary>
+        /// Instance of the converter with settings:
+        /// <see cref="ConvertNullValue"/> is <see langword="true"/>, <see cref="TrimStringValue"/> is <see langword="true"/>.
+        /// </summary>
+        public static readonly NullAndTrimStringConverter ConvertNullAndTrimString = new NullAndTrimStringConverter(true, true);
+
+        internal NullAndTrimStringConverter(bool convertNullValue, bool trimStringValue)
```

Comment:
A converter cannot contain a parametric constructor if we want to use it with the `ConverterAttribute`. Fortunately, fluent definition supports setting a specific instance of converter, not just type (I was prudent 🙂).
I'm OK with that, I'm just noticing.